pandas/core/arrays/period.py

```diff
@@ -1089,3 +1111,38 @@ def _make_field_arrays(*fields):
     ]
 
     return arrays
+
+
+if _PYARROW_INSTALLED and (
+    LooseVersion(pyarrow.__version__) >= LooseVersion("0.14.1.dev")
+):
+
+    class PeriodType(pyarrow.ExtensionType):
```

Comment:
Can you call this PyarrowPeriodType? I briefly thought you were redefining PeriodDtype :)

Reply:
Yeah, that's probably a good idea :)